pegasus/sites.v3/code.org/views/inspirational_videos.haml

```diff
@@ -16,4 +16,4 @@
 %div{style: 'clear: both'}
 %br/
 -if request.locale == 'en-US'
-  %p= I18n.t(:inspirational_videos_more)
+  %p{style: 'padding: 10px'}= I18n.t(:inspirational_videos_more)
```

Comment:
I thought it would be kinda silly to include a css file for just 1 attribute. It would either be a css file with just 1 attribute (lol) or I would add it to a general pegasus css file and I don't think all of that needs to be included again.